modules/monitoring-understanding-the-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,32 @@
+= Understanding the Cluster Observability Operator
+
+A default monitoring stack created by the Cluster Observability Operator (COO) includes a highly available Prometheus instance capable of sending metrics to an external endpoint by using remote write.
+
+Each COO stack also includes an optional Thanos Querier component, which you can use to query a highly available Prometheus instance from a central location, and an optional Alertmanager component, which you can use to set up alert configurations for different services.
+
+[id="advantages-of-using-cluster-observability-operator_{context}"]
+== Advantages of using Cluster Observability Operator
+
+The `MonitoringStack` CRD used by the COO offers an opinionated default monitoring configuration for COO-deployed monitoring components, but you can customize it to suit more complex requirements.
+
+Deploying a COO-managed monitoring stack can help meet monitoring needs that are difficult or impossible to address by using the core platform monitoring stack deployed by the Cluster Monitoring Operator (CMO).
+A monitoring stack deployed using COO has the following advantages over core platform and user workload monitoring:
+
+* *Extendability*. Users can add more metrics to a COO-deployed monitoring stack, which is not possible with core platform monitoring without losing support.
+In addition, COO-managed stacks can receive certain cluster-specific metrics from core platform monitoring by using federation.
+* *Multi-tenancy support*. The COO can create a monitoring stack per user namespace.
+You can also deploy multiple stacks per namespace or a single stack for multiple namespaces.
+For example, cluster administrators, SRE teams, and development teams can all deploy their own monitoring stacks on a single cluster, rather than having to use a single shared stack of monitoring components.
+Users on different teams can then independently configure features such as separate alerts, alert routing, and alert receivers for their applications and services.
+* *Scalability*. You can create COO-managed monitoring stacks as needed.
+Multiple monitoring stacks can run on a single cluster, which can facilitate the monitoring of very large clusters by using manual sharding. This ability addresses cases where the number of metrics exceeds the monitoring capabilities of a single Prometheus instance.
+* *Flexibility*. Deploying the COO with the Operator Lifecycle Manager (OLM) decouples COO releases from {product-title} release cycles.
+This method of deployment enables faster release iterations and the ability to respond rapidly to changing requirements and issues.
+Additionally, by deploying a COO-managed monitoring stack, users can manage alerting rules independently of {product-title} release cycles.
+* *Highly customizable*. The COO can delegate ownership of single configurable fields in custom resources to users by using server-side apply (SSA), which enhances customization.
```

Comment:
```suggestion
* *Highly customizable*. The COO can delegate ownership of single configurable fields in custom resources to users by using Server-Side Apply (SSA), which enhances customization.
```
Seems like it's init-capped "Server-Side Apply" in the reference Kube doc, FWIW: https://kubernetes.io/docs/reference/using-api/server-side-apply/

Reply:
Fixed.